frontend/app/src/components/helper/NftDetails.vue

```diff
@@ -20,8 +20,11 @@
           />
         </template>
       </div>
-      <div class="ml-4" :class="css.name">
-        {{ name }}
+      <div class="ml-5">
+        <div v-if="name" class="font-weight-medium" :class="css.name">
+          {{ name }}
+        </div>
+        <v-skeleton-loader v-else class="mt-1" width="100" type="text" />
```

Comment:
I have not seen any NFT without a collection name so far but with this approach, if the name/collection is empty or missing or null the loader will not go away correct?

Reply:
Hmm okay I think I was wrong assuming that the collection is always exist.
I think I need to check the loading status to show the skeleton
and show the identifier if loading is done and no name/collection provided (?)

Reply:
Yes, btw do you think when we fall back to the NFT identifier, instead of showing it as it is, maybe we can replace the `_` with whitespace ` ` and make the address truncated and a link to etherscan? So that it shows like `NFT 0x000000 1234` where the address is clickable and it takes you to the the explorer page.

Reply:
hmm okay let me see what can I do